io.github.paledega.alpine-rootfs.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: io.github.paledega.alpine-rootfs
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: alpine
+finish-args:
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why do you need this dbus permission?

Reply:
alpine rootfs terminal is only can created with `flatpak-spawn --host bwrap ...` command.

bwrap+proot require a host command call. (Because flatpak environment broke proot's fakeroot feature.)

Also alpine is actually a linux distribution and some applications require dbus.

Reply:
```bash
exec flatpak-spawn --host --clear-env bwrap --bind ${DESTDIR}/ / \
        --bind / /host \
        --proc /proc \
        --bind /sys /sys \
        --dev /dev \
        --bind /tmp /tmp \
        --share-net \
        --die-with-parent \
        --hostname alpine \
        --dir /run/user/ \
        --unshare-uts \
        --dev-bind /dev/dri /dev/dri \
        /proot -w / -0 -r / /bin/sh -c "
            export PULSE_SERVER=127.0.0.1
            export USER=root
            export DISPLAY=:0
            . /etc/profile
            exec /bin/bash
        "
```

flatpak-spawn --host : run command on host.
bwrap : for more isolation
proot : fakeroot environment like termux

Reply:
Unresolving. [Atoms](https://beta.flathub.org/apps/details/pm.mirko.Atoms) work without it.

Reply:
Thanks. I removed flatpak-spawn and permissions. Now running without bus permission.

Also I replaced rootfs extract location. so we dont need home binding...